File: Client/src/app/shared/models/Ingredient.ts
export type Ingredient = {
    id: number;
    name: string | null;
    imageUrl: string | null;
    isVegan: boolean;
    isVegetarian: boolean;
    kcalPer100g: number;
    proteinPer100g: number;
    carbsPer100g: number;
    fatPer100g: number;
}

File: Models/Entities/Ingredient.cs
namespace Models.Entities
{
    public class Ingredient
    {
        public int Id { get; set; }
        public string? Name { get; set; }
        public string? ImageUrl { get; set; }

        public bool IsVegan { get; set; }
        public bool IsVegetarian { get; set; }

        public double KcalPer100g { get; set; }
        public double ProteinPer100g { get; set; }
        public double CarbsPer100g { get; set; }
        public double FatPer100g { get; set; }

    }
}
